man/systemd.special.xml

```diff
@@ -503,8 +514,11 @@
       <varlistentry>
         <term><filename>sysinit.target</filename></term>
         <listitem>
-          <para>A special target unit covering early boot-up
-          scripts.</para>
+          <para>This target pulls in the services required for system
+          initialization. System services pulled in by this target should
+          be prepared to work without all mount points, and should declare
```

Comment:
I'd like this man page to tell me as a user/developer what sysinit means, rather than rewrite sysinit.target in prose (I freely admit I may be failing at this). While local-fs.target and swap.target are brought in by sysinit.target, I don't see how the fact that local-fs.target is pulled in is important, since if I need any non-root paths in my service I still need to add them to my dependencies.

I suppose I could add a note saying that these targets are brought in, or maybe even better add a reference to bootup(7) both here and in basic.target so as to not duplicate information. Something like the following:

> This target pulls in the services required for system initialization. System services pulled in by this target should declare DefaultDependencies=no and specify all its dependencies manually, including non-root filesystem paths.
> For details on the dependencies involved refer to bootup(7)

If that's ok with you I'll try to make something along these lines but readable.